ftw_tools/cli.py

```diff
@@ -284,11 +284,13 @@ def scene_selection(year, cloud_cover_max, bbox, buffer_days, out):
     help="Bounding box to use for the download in the format 'minx,miny,maxx,maxy'",
 )
 @click.option(
-    "--use_mcp",
-    is_flag=True,
-    help="Use Microsoft Planetary Computer to download the images. Defaults to True. Earth search used if set to false.",
+    "--stac_host",
```

Comment:
```suggestion
    "--stac_host",
    "-h",
```

For the lazy people :-)